<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="title">
     <string>文件（&amp;F）</string>
    </property>
    <addaction name="NewAction"/>
    <addaction name="SaveAction"/>
    <addaction name="OpenAction"/>
    <addaction name="DelAction"/>
   </widget>
   <widget class="QMenu" name="menu_E">
    <property name="title">
     <string>编辑（&amp;E）</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_V">
    <property name="title">
     <string>视图(&amp;V)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_B">
    <property name="title">
     <string>构建(&amp;B)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_D">
    <property name="title">
     <string>调试(&amp;D)</string>
    </property>
    <addaction name="actionDeBug"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_E"/>
   <addaction name="menu_V"/>
   <addaction name="menu_B"/>
   <addaction name="menu_D"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="NewAction">
   <property name="text">
    <string>新建(&amp;N)</string>
   </property>
  </action>
  <action name="SaveAction">
   <property name="text">
    <string>保存(&amp;S)</string>
   </property>
  </action>
  <action name="OpenAction">
   <property name="text">
    <string>打开(&amp;O)</string>
   </property>
  </action>
  <action name="DelAction">
   <property name="text">
    <string>删除(&amp;X)</string>
   </property>
  </action>
  <action name="actionDeBug">
   <property name="text">
    <string>DeBug</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
